tests/Test/Pager/Subscriber/Filtration/Doctrine/ORM/QueryTest.php

```diff
@@ -312,7 +312,14 @@ public function shouldFilterNumericFilterValuesOne(): void
         $this->assertEquals(2, $this->queryAnalyzer->getNumExecutedQueries());
         $executed = $this->queryAnalyzer->getExecutedQueries();
 
-        $this->assertEquals('SELECT a0_.id AS id_0, a0_.title AS title_1, a0_.enabled AS enabled_2 FROM Article a0_ WHERE a0_.title = 1 LIMIT 10', $executed[1]);
+        // Different aliases separators according to Doctrine version
+        if (version_compare(\Doctrine\ORM\Version::VERSION, '2.5', '<')) {
```

Comment:
This is not needed anymore: we don't support Doctrine before 2.7

Reply:
Done.  I've removed the version check and associated code for the old version of Doctrine.